### ISSUE TITLE
Compatibility with ember-legacy-views

### ISSUE THREAD

**Issue by rmachielse** [ember-cli/ember-cli-htmlbars#71]
Since 1.0.4 I am seeing the following error when running `ember serve`:

```
Error: Assertion Failed: Using `{{view}}` or any path based on it ('app/templates/dashboard.hbs' @ L14:C10) has been removed in Ember 2.0
```

I do have the `ember-legacy-views` addon installed.
I don't see this anymore when downgrading to `ember-cli-htmlbars` 1.0.3

**Comment by rwjblue** [comment]
Thanks for reporting!  Which Ember version are you using?

**Comment by rmachielse** [comment]
@rwjblue sure thing! I've tested this with Ember 2.4.5 as well as 2.5.0, there doesn't seem to be a difference between those two. I inspected the `ENV` variable in [this](https://github.com/emberjs/ember.js/blob/master/packages/ember-template-compiler/lib/plugins/assert-no-view-helper.js#L17) location in the build process and it seems to be empty there.

**Comment by rwjblue** [comment]
OK, awesome.  This is definitely due to the changes in https://github.com/ember-cli/ember-cli-htmlbars/pull/70.  I will dig in shortly, sorry about the troubles!

**Comment by rmachielse** [comment]
Thanks! No problem, for now a downgrade to 1.0.3 seems to be fine

**Comment by rwjblue** [comment]
I just revertted and released v1.0.5 for now, I will revisit the original thing I was solving in #70 later...
